CHANGES.rst

```diff
@@ -1,5 +1,8 @@
 CHANGELOG
 =========
+0.3
+---
+- Add canned responses
```

Comment:
@maltem-za can we add the change log when we do the release, and remove it from this PR